types/get-node-text.d.ts

```diff
@@ -1 +1,2 @@
-export function getNodeText(node: HTMLElement): string;
+declare function getNodeText(node: HTMLElement): string;
+export { getNodeText };
```

Comment:
Same here. Do we need `.d.ts` files when the source is in TypeScript?

Reply:
shouldn't need to. Might need to add the `-d` flag when compiling or set that in your tsconfig.